manuscript/01.introduction.md

```diff
@@ -6,15 +6,18 @@ One of the primary treatments of cancer consists of chemotherapy, mainly targete
+<!-- Lineage data is special -->
 
-<!-- literature review of survivorship and time censoring -->
-There are two difficulties associated with single-cell lineage data. In each lineage tree, edges are representative of the time duration that takes a particular cell to divide. The clades that contain cells that divide faster are more represented in the population and these overrepresented colonies introduce survivorship bias to the sample data. If this bias is disregarded, the model would fail to converge to the correct answer. In some studies the survivorship bias is addressed by adopting different strategies to sample from the lineage trees such as forward sampling, where starting from the root cell they track cells over generations by randomly selecting a daughter cell at each division, retrospective sampling which traverses the tree from a leaf cell to the root cell, and tree sampling where they removed leaf cells [@doi:10.1101/488981; @doi:10.3389/fphy.2018.00064]. Another difficulty is the time censorship. At the final time point of an experiment, cells that are still alive will die/divide at some point after the cut off time of the experiment and inevitably results in missing information about their lifetime and fate. On the other hand, there are cells that have been born at an unkonwn time before the beginning of the experiment. In thses two cases, due to the finite experiment duration the data is right- and left-censored, respectively. Time censoreship needs to be properly handled such that censored values are treated differently from uncensored values. In the process of fitting, we intend to find the most likely distribution source of the random values, i.e., measured phenotypes. To address the time censoreship issue, Kuchen and colleauges truncated the lineages after the last generations [@doi:10.7554/eLife.51002]. We employed the survival distribution function specifically for censored samples in the data and fitted the original probability distributions to uncensored samples, which resulted in accurate and correct convergence of the model.  
+Measurements accompanied by lineage relationships are uniquely valuable for studying inherited phenotypes within families of people or populations of cells. This value is well-recognized in linkage studies which use pairs of relatives to identify the genetic determinants of disease [@pmid:19136655]. Notably, linkage studies can identify genetic determinants with greater power than even much larger correlation-based studies because relatives essentially serve as internal controls (CITE REVIEW). Linkage studies also start with the phenotype of individuals, rather than grouping based on molecular differences as is most common in cells. Lineage-resolved data has demonstrated unique value in cells for uncovering heterogeneity due to transient differences [@doi:10.1101/373258; @DOI:10.1073/pnas.1715639115].
```

Comment:
Are either of these two suitable for the reference here?
[Kuchen et al](https://www.biorxiv.org/content/10.1101/373258v1.abstract)
[Chao et al](https://www.embopress.org/doi/full/10.15252/msb.20188604)

Reply:
I do reference Kuchen et al. I'm not sure that Chao really demonstrates the value of having lineage relationships, but we should definitely reference it when we bring up the gamma-distributed times.